### PR TITLE
Update pr template

### PR DESCRIPTION
**Pull request by g1eny0ung**
### What problem does this PR solve?

> Problem Summary:

Some fields are outdated in the PR template.

### What's changed and how it works?

> What's Changed:

- Remove repo `chaos-mesh/website-zh` in `Related changes` because it has already been archived
- Make `Close #xxx` and `Proposal: [xxx](url)` be commented initially because in most cases we don't need to fill them (In my experience)
- Use checkbox list in `Related changes` (Default **NO** if you aren't filled)

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Chaos Dashboard`
- [ ] Need to **cheery-pick to release branches**

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
 <!--
 Thank you for contributing to Chaos Mesh!
 
-If you haven't already, please read Chaos Mesh's [CONTRIBUTING](https://github.com/chaos-mesh/chaos-mesh/blob/master/CONTRIBUTING.md) document.
+If you're not sure how to do it, please read the CONTRIBUTING document of Chaos Mesh first.
```
Comment (milasuperstar) [inline .github/pull_request_template.md:4]:
```suggestion
If you're unsure where to start, please refer to the contributing doc:
```

Comment (milasuperstar) [reply .github/pull_request_template.md:4]: Rest LGTM

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,42 +1,43 @@
...
 
 ### What problem does this PR solve?
 
-<!--
-Automatically closes linked issue when PR is merged.
-Usage: `close #<issue number>`
--->
-Issue Number: close #xxx <!-- REMOVE this line if no issue to close -->
+<!-- Uncomment this line if some issues to close -->
+<!-- Close #xxx -->
```
Comment (iguoyr) [inline .github/pull_request_template.md:19]: I think `#<issue number>` is better.

Comment (g1eny0ung) [reply .github/pull_request_template.md:19]: Sounds good. 💯 I will update it soon.

Comment (g1eny0ung) [reply .github/pull_request_template.md]: Done in dd040e5.

**Review by STRRL — APPROVED** [review]
LGTM

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**